independent-projects/arc/pom.xml

```diff
@@ -190,6 +190,11 @@
   </build>
 
   <repositories>
+    <repository>
```

Comment:
could you remove this? I'm trying to get rid of <repository> tags so this is going into the wrong direction :)

Reply:
So for this one:
- we need it as long as we have JBoss Nexus declared to be sure we try Central first;
- we can remove it as soon as the JBoss Nexus repo is deleted.

Reply:
Ok so if you merge my other PR already this can get removed?

Reply:
Exactly. And done!